src/external/CL/permission

```diff
@@ -63,7 +63,7 @@ header files with an open source project called "Darktable".
 Darktable is a photography workflow solution for Linux with a virtual=20
 lighttable and darkroom. Its main focus and strength is in conversion and=20
 development of photographic RAW-files. Details can be found at=20
-http://darktable.sourceforge.net. Darktable is distributed in source code a=
```

Comment:
Pretty sure you should not be changing this.

Reply:
You are right.